lib/rides/pages/search_ride_page.dart

```diff
@@ -51,8 +51,6 @@ class _SearchRideFormState extends State<SearchRideForm> {
     super.initState();
     _firstDate = DateTime.now();
     _selectedDate = DateTime.now();
-    _dateController.text = FormatHelper.formatDate(_selectedDate);
-    _timeController.text = FormatHelper.formatTime(_selectedDate);
```

Comment:
if we remove this here shouldn't we set it at the start of the build method like for Create_Drive because now we don't have default values for Date and Time

Reply:
oh I put at the top of showTimepicker, but I wanted to put it at the top of buildTimepicker. Nice catch!